assets/js/modules/adsense/util/site-stats-data.js

```diff
@@ -49,11 +61,58 @@ export function getSiteStatsDataForGoogleChart( current, previous, label, select
 
 	const ends = stringToDate( current.endDate );
 
+	const locale = getLocale();
+	const localeDateOptions = {
+		weekday: 'short',
+		month: 'short',
+		day: 'numeric',
+	};
+
 	while ( +currentDate <= +ends ) {
+		const currentMonth = parseFloat( ( current?.rows || [] ).find( findRowByDate( currentDate ) )?.[ selectedColumn ] || 0 );
+		const prevMonth = parseFloat( ( previous?.rows || [] ).find( findRowByDate( previousDate ) )?.[ selectedColumn ] || 0 );
+
+		const difference = prevMonth !== 0
+			? ( currentMonth * 100 / prevMonth ) - 100
+			: 100; // if previous month has 0, we need to pretend it's 100% growth, thus the "difference" has to be 100
+
+		const dateRange = sprintf(
+			/* translators: %1$s: date for user stats, %2$s: previous date for user stats comparison */
+			_x( '%1$s vs %2$s', 'Date range for AdSense dashboard chart tooltip', 'google-site-kit' ),
+			currentDate.toLocaleDateString( locale, localeDateOptions ),
+			previousDate.toLocaleDateString( locale, localeDateOptions ),
+		);
+
+		let tooltipData = readableLargeNumber( currentMonth, metadata?.currency );
+		if ( metadata?.type === 'METRIC_RATIO' ) {
+			tooltipData = numberFormat( currentMonth, { style: 'percent' } );
+		}
+
+		const statInfo = sprintf(
+			/* translators: %1$s: selected stat label, %2$s: numberic value of selected stat, %3$s: up or down arrow , %4$s: different change in percentage, %%: percent symbol */
+			_x( '%1$s: <strong>%2$s</strong> <em>%3$s %4$s%%</em>', 'Stat information for AdSense dashbaord chart tooltip', 'google-site-kit' ),
```

Comment:
See above, let's adjust the context for this string and the similar one for Analytics so that they use the same context.